### ISSUE TITLE
Add new feature assembling strategy

### ISSUE THREAD

**Issue by chicxurug** [sed-inf-u-szeged/DeepWaterFramework#20]
Create a new "Manual File Input with Fixed Test Set" strategy, where one can use fixed training and fixed test csv pairs. By default only one pair needs to be set up, but the user can add multiple csv pairs as well. In this case the training won't use 10-fold cross-validation but use the provided train-test csv pairs (if 5 pairs were given, 5-fold will be performed). In this case the models should be dumped as well (pickle for scikit and dump for tensorflow). Dump models in each fold. In a sense, this feature assembling will be the generalization of the automatic 10-fold cross-validation.
